docs/cog_guides/audio.rst

```diff
@@ -0,0 +1,3741 @@
+.. _audio:
+
+=====
+Audio
+=====
+
+This is the cog guide for the audio cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this:
+
+    	[p]load audio
+
+
+You can see additional help for any command in this guide by using ``[p]help`` with the
+command name, like ``[p]help playlist append``.
+
+In this guide, you will see references to "Lavalink" or the "Lavalink.jar". `Lavalink <https://github.com/freyacodes/Lavalink/>`_ is the
+Java-based audio backend we use to be able to play music through the bot. Most users will
+not have to worry much about Lavalink or what it is, as Audio manages this process for you
+by default. Advanced users can read more about Lavalink and special cases under the 
+:ref:`Lavalink - Advanced Usage<advanced-usage>` section below.
+
+You will also see references to ``managed`` or ``unmanaged`` in regards to the Lavalink.jar.
+The default is ``managed``, as Audio manages the Lavalink.jar in this state. If it is run as an
+``unmanaged`` process, this means you as the bot owner will be managing the Lavalink process yourself.
+
+When the Audio cog is first loaded, it will contact GitHub and download the newest Lavalink.jar file
+for your bot to use. Any time Red is updated, and you update your bot, most likely it will download a new,
+updated Lavalink.jar for your bot. It is important to keep your bot updated as fixes are provided for Lavalink frequently.
+
+All commands should be used in a Discord server, and not in Direct Messages as Audio is not available there.
```

Comment:
Discord server - as opposed to another kind of server?

Reply:
The docs reference other server type so it makes sense reintroducing the discord context here